test/std-test.cc

```diff
@@ -46,6 +48,25 @@ TEST(ranges_std_test, format_vector_path) {
 #endif
 }
 
+TEST(ranges_std_test, format_quote_path) {
+  // Test quote problem described in #3319.
```

Comment:
Since the issue is very simple I suggest summarizing it instead of linking elsewhere, e.g. "Test that path is not escaped twice in the debug mode."

Reply:
Done.